pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarSplit.java

```diff
@@ -95,7 +97,23 @@ public PulsarSplit(
         this.tupleDomain = requireNonNull(tupleDomain, "tupleDomain is null");
         this.startPosition = PositionImpl.get(startPositionLedgerId, startPositionEntryId);
         this.endPosition = PositionImpl.get(endPositionLedgerId, endPositionEntryId);
+        this.schemaInfoProperties = schemaInfoProperties;
         this.offloadPolicies = offloadPolicies;
+
+        SchemaInfo schemaInfoTemp;
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            schemaInfoTemp = SchemaInfo.builder()
+                    .name(originSchemaName)
+                    .type(schemaType)
+                    .schema(schema.getBytes("ISO8859-1"))
+                    .properties(objectMapper.readValue(schemaInfoProperties, Map.class))
+                    .build();
+        } catch (Exception e) {
+            log.error("Create schemaInfo failed!", e);
+            schemaInfoTemp = SchemaInfo.builder().build();
```

Comment:
If exception cause when reading schema properties, we'd better throw a runtime exception.